clab/export.go

```diff
@@ -55,8 +64,13 @@ func (c *CLab) exportTopologyDataWithTemplate(_ context.Context, w io.Writer, p
 				a, _ := json.MarshalIndent(v, prefix, indent)
 				return string(a)
 			},
-		}).
-		ParseFiles(p)
+		})
+	var err error
+	if len(p) != 0 {
+		_, err = t.ParseFiles(p)
+	} else {
+		_, err = t.Parse(defaultExportTemplate)
+	}
```

Comment:
To let people use the embedded full.tmpl config, let's say that if a user provides a string value `__full` as the template path, this would trigger the usage of the full.tmpl